radicle-httpd/src/raw.rs

```diff
@@ -27,12 +29,85 @@ async fn file_handler(
 ) -> impl IntoResponse {
     let storage = &profile.storage;
     let repo = Repository::open(paths::repository(storage, &project))?;
+    let mut response_headers = HeaderMap::new();
+
+    if repo.file(sha, &path)?.content(&repo)?.size() > MAX_BLOB_SIZE {
+        return Ok::<_, Error>((StatusCode::PAYLOAD_TOO_LARGE, response_headers, vec![]));
+    }
+
     let blob = repo.blob(sha, &path)?;
+    let mime = {
+        if let Some(ext) = path.split('.').last() {
+            match ext {
+                "aac" => "audio/aac",
+                "avi" => "video/x-msvideo",
+                "bin" => "application/octet-stream",
+                "bmp" => "image/bmp",
+                "bz" => "application/x-bzip",
+                "bz2" => "application/x-bzip2",
+                "csh" => "application/x-csh",
+                "css" => "text/css",
+                "csv" => "text/csv",
+                "doc" => "application/msword",
```

Comment:
list taken from mozilla [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types)

we can create a static map but needs [phf](https://crates.io/crates/phf), and if we're adding a crate, then I suggest `mime_guess`

Reply:
actually, made it static [here](https://github.com/radicle-dev/heartwood/compare/a4ddfee881c5c76c07c473501a605d1d545580ad..08b132b5341b93953ab0ed91c0d2c84b02bfa38b) without importing anything